### PR TITLE
Streamlined checklists, added highlighting where possible.

### PR DESCRIPTION
**Pull request by DoToAdventures**
**Summary of Changes**

Streamlined the checklists and made correction according to the new SOP published in the discord.
I also added instrument highlighting where possible without modifying modelbehaviourdefs.

**Screenshots (if necessary)**

https://i.imgur.com/NcQ3mN1.png

**Additional context**

TODO for checklists: Add PartIDs for instruments in modelbehaviourdefs where possible to implement highlighting for most instruments. After that I will work on evaluation of checklist items through simvars.

### CODE REVIEW

File: A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/Checklist/Airbus_A320neo_Checklist.xml [hunk 1]
```diff
@@ -29,7 +28,6 @@
 				<Checkpoint ReferenceId="APU_Bleed_On"/>
 				<Checkpoint ReferenceId="A320_PREP_CHECK_OVRHEAD"/>
 				<Checkpoint ReferenceId="A320_PREP_XBLEED"/>
-				<Checkpoint ReferenceId="A320_PREP_WEATHER_RADAR"/>
```
Comment (Venorcis) [inline A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/Checklist/Airbus_A320neo_Checklist.xml:32]: I would keep this one? Removing the RPM3 & Center Tank Pump is indeed fine (I'd personally even remove them from the Library.xml; it's in version history anyway)

File: A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/Checklist/Library.xml [hunk 2]
```diff
@@ -1,1275 +1,1360 @@
...
+         <CheckpointDesc SubjectTT="TT:Engine Master Switches" ExpectationTT="TT:ALL OFF" />
+         <Instrument Id="ENGINE_Switch_Master1" />
+         <Instrument Id="ENGINE_Switch_Master2" />
+      </Checkpoint>
+      <Checkpoint Id="A320_PREP_ENGINE_MODE_SELECTOR">
+         <CheckpointDesc SubjectTT="TT:Engine Mode Selector" ExpectationTT="TT:GAME.CHECKLIST_NORM" />
+         <Instrument Id="ENGINE_Switch_Engine_Mode" />
+      </Checkpoint>
+      <Checkpoint Id="A320_PREP_LANDING_GEAR_LEVER">
+         <CheckpointDesc SubjectTT="TT:Landing Gear Lever" ExpectationTT="TT:Check DOWN position" />
```
Comment (Venorcis) [inline A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/Checklist/Library.xml:18]: "CHECK DOWN"  might be better as expectation?

File: A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/Checklist/Library.xml [hunk 3]
```diff
@@ -1,1275 +1,1360 @@
...
+         <CheckpointDesc SubjectTT="TT:FMGS: ATC Clearance" ExpectationTT="TT:OBTAIN" />
+      </Checkpoint>
+      <Checkpoint Id="A320_CPREP1_FMGS_GROSS_WEIGHT">
+         <CheckpointDesc SubjectTT="TT:FMGS: Gross Weight Insertion" ExpectationTT="TT:CHECK" />
+      </Checkpoint>
+      <Checkpoint Id="A320_CPREP1_FMGS_TO_DATA_CHECK">
+         <CheckpointDesc SubjectTT="TT:FMGS: Takeoff Data" ExpectationTT="TT:CALCULATE/CHECK" />
+      </Checkpoint>
+      <Checkpoint Id="A320_CPREP1_FMGS_FPLAN_AB_CHECK">
+         <CheckpointDesc SubjectTT="TT:FMGS: Flight Plan A and B" ExpectationTT="TT:Check" />
```
Comment (Venorcis) [inline A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/Checklist/Library.xml:378]: Expectation CHECK (uppercase)?

File: A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/Checklist/Library.xml [hunk 4]
```diff
@@ -1,1275 +1,1360 @@
...
+      <Checkpoint Id="A320_BeforeTakeoff_PACKS">
+         <CheckpointDesc SubjectTT="TT:Packs (INOP)" ExpectationTT="TT:AS REQUIRED" />
+      </Checkpoint>
+      <Checkpoint Id="A320_BeforeTakeoff_EXT_LIGHT">
+         <CheckpointDesc SubjectTT="TT:Exterior Light" ExpectationTT="TT:SET" />
+      </Checkpoint>
+      <Checkpoint Id="A320_BeforeTakeoff_SLIDING_TABLE">
+         <CheckpointDesc SubjectTT="TT:Sliding Table (INOP)" ExpectationTT="TT:STOWED" />
+      </Checkpoint>
+      <Checkpoint Id="A320_BeforeTakeoff_ATC_CLRD_TO">
```
Comment (Venorcis) [inline A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/Checklist/Library.xml:599]: spurious ) in expectation

File: A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/Checklist/Library.xml [hunk 5]
```diff
@@ -1,1275 +1,1360 @@
...
+         <Instrument Id="ENGINE_Switch_Engine_Mode" />
+      </Checkpoint>
+      <Checkpoint Id="A320_AFTERLANDING_ATC">
+         <CheckpointDesc SubjectTT="TT:ATC" ExpectationTT="TT:STBY/OFF" />
+      </Checkpoint>
+      <Checkpoint Id="A320_AFTERLANDING_TCAM_MODE_SEL">
+         <CheckpointDesc SubjectTT="TT:TCAM Mode Selector (INOP)" ExpectationTT="TT:STBY" />
+      </Checkpoint>
+      <Checkpoint Id="A320_AFTERLANDING_RADAR">
+         <CheckpointDesc SubjectTT="TT:Radar" ExpectationTT="TT:OFF/STBY" />
```
Comment (Venorcis) [inline A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/Checklist/Library.xml:1147]: I changed this to Weather Radar with expectation OFF (there is no STBY?)

Comment (DoToAdventures) [reply A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/Checklist/Library.xml:1147]: SYS on with Mode Off = STBY perhaps?

File: A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/Checklist/Airbus_A320neo_Checklist.xml (hunk 1 above)
Comment (DoToAdventures) [reply A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/Checklist/Airbus_A320neo_Checklist.xml:32]: Not in the SOP at that point, hence removed.

File: A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/Checklist/Library.xml (hunk 5 above)
Comment (Venorcis) [reply A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/Checklist/Library.xml:1147]: The OFF mode is INOP sadly, but that could be the case indeed. Just above is STDBY/OFF so it feels a bit inconsistent, and the Radar > Weather Radar change is nice to do everywhere consistently I think though :)

File: A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/Checklist/Airbus_A320neo_Checklist.xml (hunk 1 above)
Comment (Venorcis) [reply A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/Checklist/Airbus_A320neo_Checklist.xml:32]: Where did you check this? Can you maybe add a URL in the comments to a reference? I could only find various quite different versions of company-specific checklists.

Comment (DoToAdventures) [reply A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/Checklist/Airbus_A320neo_Checklist.xml:32]: https://cdn.discordapp.com/attachments/740273811617874043/752276022254370847/A32NX_Project_Standard_Operating_Manual_1.pdf Not 100% consistent with this right now but it should for the most part say the same stuff.

**Comment by Venorcis** [comment]
Nice! Awesome working with you @DoToAdventures, from my perspective this is ready to go now :)

**Pull request merged**